sftp/1.3/modules/ROOT/pages/sftp-documentation.adoc

```diff
@@ -558,10 +558,19 @@ You can disable this feature by omitting a value.  |  |
 
 * WHEN_EXHAUSTED_GROW
 * WHEN_EXHAUSTED_WAIT
-* WHEN_EXHAUSTED_FAIL a| Specifies the behavior of the Mule component pool when the pool is exhausted. Possible values are:
+* WHEN_EXHAUSTED_FAIL a| Specifies the behavior of the Mule component pool when the pool is exhausted:
 
-* `WHEN_EXHAUSTED_FAIL`, which will throw a `NoSuchElementException`. * `WHEN_EXHAUSTED_WAIT`, which will block by invoking `Object.wait(long)` until a new or idle object is available.
-* `WHEN_EXHAUSTED_GROW`, which will create a new Mule instance and return it, essentially making `maxActive` meaningless. If a positive `maxWait` value is supplied, it will block for, at most, the specified number of milliseconds, after which a `NoSuchElementException` will be thrown. If `maxThreadWait` is a negative value, it will block indefinitely. |  |
+* `WHEN_EXHAUSTED_FAIL`
++
+Throws a `NoSuchElementException` error.
+
+* `WHEN_EXHAUSTED_WAIT`
++
+Blocks by invoking `Object.wait(long)` until a new or idle object is available. If *Max Wait* is a positive value, it blocks for, at most, the specified number of milliseconds, after which a `NoSuchElementException` error is thrown. If *Max Wait* is a negative value, it blocks indefinitely.
```

Comment:
Do you know what is doing the blocking (to make this more active)? if not, then leave what you have.